src/jit/emitxarch.cpp

```diff
@@ -5535,15 +5535,28 @@ static bool isSseShift(instruction ins)
     }
 }
 
-static bool isSSEExtract(instruction ins)
+//------------------------------------------------------------------------
+// IsDstSrcImmAvxInstruction: check if instruction has RM R I format
+// for all encodings: EVEX, VEX and legacy SSE
```

Comment:
> instruction has RM R I format

This comment is not correct, certain instructions can be "ins R RM I". 
BTW, I still think the comment should emphasize "NO VEX.NDS".

Reply:
@fiigii Can I correct it in next PR. I need `Ssse2.Shuffle` to implement some remaining SSE2 helpers and I am blocked to some extent on this PR.

Reply:
Not sure, this PR is still waiting for MS maintainers' final review.

> I need Ssse2.Shuffle to implement some remaining SSE2 helpers and I am blocked to some extent on this PR.

Actually, you can copy the one-line definition of Shuffle into your current workspace, and rebase after this PR gets merged.